classes/patterns/class-campaign.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Campaign pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+use P4GBKS\Patterns\Templates\Covers;
+
+/**
+ * Class Campaign.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Campaign extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/campaign-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$classname  = self::get_classname();
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+
+		return [
+			'title'      => __( 'Campaign', 'planet4-blocks-backend' ),
+			'blockTypes' => [ 'core/post-content' ],
+			'categories' => [ 'layouts' ],
+			'content'    => '
+				<!-- wp:group {"className":"block ' . $classname . '"} -->
+					<div class="wp-block-group block ' . $classname . '">
+						' . PageHeader::get_config()['content'] . '
+						' . SideImageWithTextAndCta::get_config( [ 'title_placeholder' => __( 'The problem', 'planet4-blocks' ) ] )['content'] . '
+						' . SideImageWithTextAndCta::get_config(
+							[
+								'title_placeholder' => __( 'The solution', 'planet4-blocks' ),
+								'media_position'    => 'right',
+								'background_color'  => 'grey-05',
+								'alignfull'         => true,
+							]
+						)['content'] . '
+						' . Covers::get_content( [ 'cover_type' => 'take-action' ] ) . '
+						<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image","backgroundColor":"grey-05"} -->
```

Comment:
This whole part will eventually be replaced by the `GravityFormWithImage` template (implemented in [this PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/918))